CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 # Changelog
 
-## [Unreleased]
+## 0.6.6
+
+**2021-11-08**
+
+**Important note:** Version v0.6.5 was accidentally tagged and should be
+avoided.  This version is identical to v0.6.4 in package structure and API, but
+has updated the version marker so that normal `go get` upgrades will not
+require modifying existing use of v0.6.4.
+
+### Version bumps (since v0.6.4)
+
+- Bump grpc from to 1.42.0.
+- Bump dgraph/badger to v2 2.2007.3.
+- Bump go.etcd.io/bbolt to 1.3.6.
+
+## 0.6.5
+
+**2021-08-04**
+
+**Important note**: This version was tagged by accident, and should not be
+used. The tag now points to the [package-reorg
+branch](https://github.com/tendermint/tm-db/tree/package-reorg) so that
+any existing dependencies should not break.
+
+## 0.6.4
```

Comment:
Please add only our changes and link to the original change log with a brief description.

Reply:
updated